quickwit/quickwit-serve/src/elastic_search_api/filter.rs

```diff
@@ -118,6 +143,22 @@ pub(crate) fn elastic_index_field_capabilities_filter() -> impl Filter<
         .and(json_or_empty())
 }
 
+#[utoipa::path(get, tag = "Metadata", path = "/_field_caps")]
+pub(crate) fn elastic_field_capabilities_filter() -> impl Filter<
```

Comment:
What is this about?

Reply:
it handles path "/_field_caps" (same as the other methods in this file)